vector/src/main/java/im/vector/activity/LoginActivity.java

```diff
@@ -1858,22 +1817,6 @@ public void onMatrixError(MatrixError e) {
     private void restoreSavedData(@NonNull Bundle savedInstanceState) {
         Log.d(LOG_TAG, "## restoreSavedData(): IN");
 
-        mLoginEmailTextView.setText(savedInstanceState.getString(SAVED_LOGIN_EMAIL_ADDRESS));
-        mLoginPasswordTextView.setText(savedInstanceState.getString(SAVED_LOGIN_PASSWORD_ADDRESS));
-        mUseCustomHomeServersCheckbox.setChecked(savedInstanceState.getBoolean(SAVED_IS_SERVER_URL_EXPANDED));
-        mHomeServerText.setText(savedInstanceState.getString(SAVED_HOME_SERVER_URL));
```

Comment:
Are you sure you want to lose the server config when the app is backgrounded on a device where the background process limit is 0?